changelog.md

```diff
@@ -1,5 +1,412 @@
+
+* Most Unicode characters should be displayed properly
+  if your fonts support them.
+  ([#1524](https://github.com/lite-xl/lite-xl/pull/1524))
+
+* LogView will no longer scroll automatically if the user had scrolled.
+  The LogView will only scroll automatically when the user scrolls up
+  to the last entry.
+  ([#1546](https://github.com/lite-xl/lite-xl/pull/1546))
+
+* When using different fonts (especially fonts that render different scripts),
+  the letters will be aligned vertically.
+  ([#1560](https://github.com/lite-xl/lite-xl/pull/1560))
+
+* Unsaved named files are now saved in the workspace.
+  ([#1597](https://github.com/lite-xl/lite-xl/pull/1597))
+
+* macOS builds are now signed with a developer certificate.
+  This allows the user to right click the application in Finder and execute
+  it directly.
+  ([#1656](https://github.com/lite-xl/lite-xl/pull/1656))
+
+### Performance Improvements
+
+* Allow command buffer to be expanded.
+  ([#1297](https://github.com/lite-xl/lite-xl/pull/1297))
+
+* Use table.move to implement common.splice.
+  ([#1324](https://github.com/lite-xl/lite-xl/pull/1324))
+
+* Create renderer only when it doesn't exist.
+  ([#1315](https://github.com/lite-xl/lite-xl/pull/1315))
+
+* Avoid drawing hidden text in DocView:draw_line_text.
+  ([#1298](https://github.com/lite-xl/lite-xl/pull/1298))
+
+* Don't calculate widths per-uft8-char when not needed.
+  ([#1409](https://github.com/lite-xl/lite-xl/pull/1409))
+
+* Allow tokenizer to pause and resume in the middle of a line.
+  ([#1444](https://github.com/lite-xl/lite-xl/pull/1444))
+
+* Optimize CI build times on MSYS2.
+  ([#1435](https://github.com/lite-xl/lite-xl/pull/1435))
+
+* Significant memory usage improvements when using huge fonts on Windows.
+  ([#1555](https://github.com/lite-xl/lite-xl/pull/1555))
+
+* The program should be much more responsive when running background tasks.
+  ([#1601](https://github.com/lite-xl/lite-xl/pull/1601))
+
+### Backward Incompatible Changes
+
+* The native plugin API is now usable on multiple source files,
+  without causing any duplicated symbol errors during compilation.
+  Plugins using the new plugin API header must define `LITE_XL_PLUGIN_ENTRYPOINT`
+  before importing the header, in one of their source files.
+  ([#1335](https://github.com/lite-xl/lite-xl/pull/1335))
+
+* The native plugin API header now follows the Lua 5.4 API.
+  Previously, the plugin API header follows the Lua 5.2 API.
+  ([#1436](https://github.com/lite-xl/lite-xl/pull/1436))
+
+* On Linux, `process.start()` will now throw an error if `execv()`` fails.
+  Previously, the function will return a valid Process object and the error message
+  can be obtained by reading the standard output.
+  ([#1363](https://github.com/lite-xl/lite-xl/pull/1363))
+
+* Lite XL will use the default `SCALE` of 1 due to unreliable display
+  scale detection. This may be fixed in a later version of Lite XL.
```

Comment:
```suggestion
* Lite XL will use the default `SCALE` of 1 due to unreliable display
  scale detection. This may be fixed in a later version of Lite XL.
  Set the `LITE_SCALE` environment variable to override this value.
```